docusaurus.config.js

```diff
@@ -191,10 +191,10 @@ module.exports = {
       copyright: `Copyright © ${new Date().getFullYear()} Camunda`,
     },
     algolia: {
-      apiKey: "f1b2a46296ae374b7f5a5c627341c354",
+      // These keys are for our new standalone algolia instance!
+      apiKey: "d701d38126d1a43866047d3ab97680d1",
```

Comment:
GitGuardian is not happy about this apikey being exposed, but it doesn't seem like there's a lot of evil that someone could do with it. Maybe the worst is to try to take down Algolia and pin it on us. 

We could take the extra time to inject it at build time, but even if we did, it'd still be exposed on the built public website.

Reply:
As you mentioned over Slack, this is the way Facebook/Docusaurus is doing it so 🤷‍♀️ .

I have no issues with this. This is working as intended.